tests/unit/test_main.py

```diff
@@ -36,53 +38,194 @@ def setup_method(self):
     async def test_create_mw_case_1(self, event_loop):
         """Test a successful case of the REST to create."""
         self.napp.controller.loop = event_loop
-        url = f'{self.base_endpoint}'
+        url = f"{self.base_endpoint}"
         start = datetime.now(pytz.utc) + timedelta(days=1)
         end = start + timedelta(hours=2)
+        self.controller.switches = MagicMock()
+        self.controller.get_interface_by_id = MagicMock()
+        self.controller.napps[("kytos", "topology")].links = MagicMock()
+        switches = {"00:00:00:00:00:00:00:02": 1, "00:00:00:00:00:00:00:03": 2}
+        interfaces = {"00:00:00:00:00:00:00:03:3": 1, "00:00:00:00:00:00:00:02:1": 2}
+        links = {
+            "cf0f4071be426b3f745027f5d22bc61f8312ae86293c9b28e7e66015607a9260": 1,
```

Comment:
Thanks for adding in the Link Testing here. Before I just ported forward the previous set of tests, expecting those to be good enough. Clearly they weren't.